models/points_test.go

```diff
@@ -1509,3 +1510,25 @@ func TestPrecisionString(t *testing.T) {
 		}
 	}
 }
+
+func Test_ParsePointsStringWithExtraBuffer(t *testing.T) {
```

Comment:
For consistency with the other tests in the file, can you remove the `_` since the others do not follow this naming convention.